advisories/a103220.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Technical Advisory 103220
+advisory: A-103220
+summary: Inserting rows into a multi-column-family table with COPY can corrupt the table, making future reads fail with internal errors.
+toc: true
+affected_versions: v23.1.0-alpha.1 to v23.1.0
+advisory_date: 2023-05-16
+docs_area: releases
+---
+Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
+
+## Description
+
+In CockroachDB v23.1.0 and its testing versions, inserting rows into a multi-[column-family](../v23.1/column-families.html) table with [`COPY`](../v23.1/copy-from.html) can corrupt the table. Future reads on the corrupted table will result in internal errors. If the table is empty before the `COPY` executes, the table must be dropped and re-[copied](../v23.1/copy-from.html) to be encoded correctly. If data was copied into a table with existing rows, the data in those rows may be irrecoverable.
```

Comment:
Its actually not entirely accurate to say "will result in internal errors" should probably be "can result in internal errors or silent data corruption"

Reply:
Updated